### DISCUSSION TITLE
Do we have a plan for the next release?

### DISCUSSION THREAD

**Discussion by orhankislal** [uber/h3#830] [General]
Hey folks,

I hope this is the appropriate channel for this question. I was wondering if there is a planned date for the next release. I am especially looking forward to using the fix from the following issue: https://github.com/uber/h3/issues/775

Thanks,

Orhan

**Comment by nrabinowitz** [comment]
This has been temporarily blocked while we work out some memory management issues that are failing our fuzzers for the new algorithm. Once these have been worked out and we're confident the new code is safe for production use we'll put out a new release.

**Reply by orhankislal** [reply]
Thanks a lot for the update!
